CHANGELOG.md

```diff
@@ -3,8 +3,9 @@ All notable changes to this project will be documented in this file.
 This project adheres to [Semantic Versioning](http://semver.org/).
 
 ## [Unreleased][unreleased]
-- 
-
+- Reverted #34 in favor of a simpler solution without breaking BC
```

Comment:
Mind using something like `Reverted PR https://github.com/Cotya/magento-composer-installer/pull/34`? 
a) It'll be easy to get the URL if people view that as text
b) `#34` is ambiguous (PR or issue)